xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-ui/src/main/resources/Refactoring/Code/RefactoringConfiguration.xml

```diff
@@ -36,7 +36,43 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content/>
+  <content>{{velocity}}
+#if ($xwiki.hasRecycleBin())
+
+#set ($configurationObject = 'Refactoring.Code.RefactoringConfiguration')
+
+{{html clean="false"}}
```

Comment:
why do you need a clean false here?

Reply:
BTW always use single quotes in velocity when you don't use variables inside. It's faster.

Reply:
@surli our rule is to always use `clean='false'`  in our code (when we can) since that's faster (no need to run the cleaner) and we control the content. I don't know if this applies here (didn't check the code)